### PR TITLE
terraform-provider-gandi: init at 0.0.1

### PR DESCRIPTION
**Pull request by manveru**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Review by matthewbauer — APPROVED** [review]

**Comment by matthewbauer** [comment]
@GrahamcOfBorg build terraform-provider-gandi

**Comment by Mic92** [comment]
We have now moved all terraform plugins into terraformPlugins: https://github.com/NixOS/nixpkgs/pull/47407

**Comment by Mic92** [comment]
It should be now possible to also move this one there.

**Comment by manveru** [comment]
@Mic92 i checked, but since their repo has no tags (or releases) it doesn't fit in with the others. I made an issue there so they'll hopefully add a tag, but until then I'll just resolve the conflict and call it a day.

File: pkgs/applications/networking/cluster/terraform-provider-gandi/default.nix [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  # if the versions are not provided via file paths.
+  postBuild = "mv go/bin/terraform-provider-gandi{,_v${version}}";
+
+  meta = with stdenv.lib; {
+    description = "Terraform provider for the Gandi LiveDNS service.";
+    homepage = "https://github.com/tiramiseb/terraform-provider-gandi";
+    license = licenses.mpl20;
+    maintainers = with maintainers; [ manveru ];
+  };
+}
```
Comment (kalbasit) [inline pkgs/applications/networking/cluster/terraform-provider-gandi/default.nix]: please move this entire package to `pkgs/applications/networking/cluster/terraform-providers/gandi`. (make sure to rebase over current master).

**Review by kalbasit — CHANGES_REQUESTED** [review]

**Comment by tiramiseb** [comment]
Hi ! I'm the author of terraform-provider-gandi. I'm familiar with Nix packages needs, I have created some Nix packages myself ;)
FYI, I will make a release very soon. It will be version 1.0.0.

**Comment by kalbasit** [comment]
oh cool! @manveru maybe wait until the release is out? FYI the update-all script does not support `deps` so you might have to introduce overrides so you can add the dependencies there.

**Comment by tiramiseb** [comment]
v1.0.0 has been released!

**Comment by zimbatm** [comment]
hey @manveru, we recently refactored how plugins are being stored

see `pkgs/applications/networking/cluster/terraform-providers`. It now includes both official and unofficial plugins.

**Comment by flokli** [comment]
Also looking very much forward to use this provider :-)

**Comment by zimbatm** [comment]
I had to rebase the branch, replacing this PR by #48223

**Pull request closed without merging**